src/NuGet/Microsoft.CodeAnalysis.Workspaces.Common.nuspec

```diff
@@ -9,15 +9,21 @@
       A shared package used by the .NET Compiler Platform ("Roslyn") including support for analyzing projects and solutions. Do not install this package manually, it will be added as a prerequisite by other packages that require it.
     </description>
     <dependencies>
-      <dependency id="Microsoft.CodeAnalysis.Common" version="[$version$]" />
-      <dependency id="Microsoft.Composition" version="$MicrosoftCompositionVersion$" />
-      <dependency id="System.Diagnostics.Contracts" version="$SystemDiagnosticsContractsVersion$" />
-      <dependency id="System.Linq.Parallel" version="$SystemLinqParallelVersion$" />
-      <dependency id="System.ObjectModel" version="$SystemObjectModelVersion$" />
-      <dependency id="System.Text.RegularExpressions" version="$SystemTextRegularExpressionsVersion$" />
-      <dependency id="System.Threading.Tasks.Parallel" version="$SystemThreadingTasksParallelVersion$" />
-      <dependency id="ManagedEsent" version="$ManagedEsentVersion$" />
-      <dependency id="Microsoft.CodeAnalysis.Elfie" version="$MicrosoftCodeAnalysisElfieVersion$" />
+      <group targetFramework="netstandard1.3">
+        <dependency id="Microsoft.CodeAnalysis.Common" version="[$version$]" />
```

Comment:
The documentation (https://docs.nuget.org/ndocs/schema/nuspec#dependencies) says:
>The <group> element without a targetFramework attribute is used as the default or fallback list of references.

My understanding of that statement is that it is not merged with any group which explicitly has a `targetFramework` specified (which would be ideal).